pandas/tests/frame/test_operators.py

```diff
@@ -972,7 +972,7 @@ def test_boolean_comparison(self):
         assert_numpy_array_equal(result, expected.values)
 
         self.assertRaises(ValueError, lambda: df == b_c)
-        self.assertFalse((df.values == b_c))
+        self.assertFalse(np.array_equal(df.values, b_c))
```

Comment:
That doesn't quite answer my question.  Is there a `non-numpy` way to assert that two `numpy` arrays (yes, those are `numpy` arrays) are not equal without using the deprecated equal operator?